src/main/java/uk/gov/hmcts/darts/common/service/impl/ObjectRecordStatusServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.hmcts.darts.common.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.darts.common.entity.ObjectRecordStatusEntity;
+import uk.gov.hmcts.darts.common.enums.ObjectRecordStatusEnum;
+import uk.gov.hmcts.darts.common.exception.CommonApiError;
+import uk.gov.hmcts.darts.common.exception.DartsApiException;
+import uk.gov.hmcts.darts.common.repository.ObjectRecordStatusRepository;
+import uk.gov.hmcts.darts.common.service.ObjectRecordStatusService;
+
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class ObjectRecordStatusServiceImpl implements ObjectRecordStatusService {
+
+    private final ObjectRecordStatusRepository objectRecordStatusRepository;
+
+    @Override
+    public ObjectRecordStatusEntity getObjectRecordStatusEntity(ObjectRecordStatusEnum objectRecordStatusEnum) {
```

Comment:
This may not be needed, there is a EodHelper that loads all the values once at startup, and can then be referenced later.